CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Replace real ORCID with example ORCID from https://orcid.org/1234-5678-9101-1121
 - Fixed the regular expression for ISBN (should only affect ISBN10 strings that didn't have any dashes or spaces, whose check digit is `X`). Issue [#323](https://github.com/citation-file-format/citation-file-format/issues/323); PR [#335](https://github.com/citation-file-format/citation-file-format/pull/335); PR [#337](https://github.com/citation-file-format/citation-file-format/pull/337)
 - Added `contributors` field. Issue [#66](https://github.com/citation-file-format/citation-file-format/issues/66), [#84](https://github.com/citation-file-format/citation-file-format/issues/84); PR [#439](https://github.com/citation-file-format/citation-file-format/pull/439)
+- Updated regex for ORCIDs to avoid trailing whitespace. Issue [#392](https://github.com/citation-file-format/citation-file-format/issues/392), PR [#467](https://github.com/citation-file-format/citation-file-format/pull/467)
```

Comment:
```suggestion
- Updated regex for ORCIDs to avoid invalid additional characters. Issue [#392](https://github.com/citation-file-format/citation-file-format/issues/392), PR [#467](https://github.com/citation-file-format/citation-file-format/pull/467)
```

As this isn't only about whitespace, this is what I'd suggest as improved wording.